content/en/integrations/guide/microsoft_teams_troubleshooting.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Troubleshooting Microsoft Teams
+description: Troubleshooting steps and solutions for issues with the Microsoft Teams integration.
+further_reading:
+- link: "/integrations/microsoft-teams/?tab=datadogapprecommended"
+  tag: "Documentation"
+  text: "Datadog Microsoft Teams Integration"
+- link: "https://app.datadoghq.com/integrations?integrationId=microsoft-teams"
+  tag: "App"
+  text: "Microsoft Teams Integration Tile"
+---
+
+## Overview
+
+This guide addresses common issues and FAQs for the Datadog Microsoft Teams integration, including missing teams, syncing channels, private channel support, user mentions, incident troubleshooting, and prompt resolution.
+
+## The team doesn't show up in the integration tile
+
+If you added the bot to the team before adding the tenant to Datadog, then Datadog would have missed the team join event to know that the team exists.
+You can try to either:
+- Synchronize your team standard channels to Datadog by running a sync command from any standard channel for that team
+- Remove the {{< region-param key="microsoft_teams_app_name" >}} app from the team, then add it back again
+
+### Synchronize your team standard channels to Datadog
+1. Navigate to a standard channel in the Team that you want to synchronize.
+2. Start a post on the channel.
+3. Type `@{{< region-param key="microsoft_teams_app_name" >}} sync`, select the `{{< region-param key="microsoft_teams_app_name" >}}` app, and then choose the `sync` command.
+4. Post the message to the channel and wait for a confirmation in the thread indicating success.
+
+### Remove and readd {{< region-param key="microsoft_teams_app_name" >}} app
+
+<div class="alert alert-danger">This removes configured connectors for that team. Perform this action only when you are ready to migrate all connectors for that team to Datadog's tenant-based integration.</div>
+
+1. Click the three dots next to the team name in the left sidebar.
```

Comment:
```suggestion
1. Click the three dots next to the Team name in the left sidebar.
```
Noticed in the above steps you capitalized Team, not sure if it needs to be here as well ? Your call though!